README.rst

```diff
@@ -144,14 +137,14 @@ is produced, something is wrong.
 By default, tests are prevented from overusing resources like disk space and
 memory.  To enable these tests, run ``make testall``.
 
-IMPORTANT: If the tests fail and you decide to mail a bug report, *don't*
+IMPORTANT: If the tests fail and you decide to file a bug report, *don't*
 include the output of ``make test``.  It is useless.  Run the failing test
 manually, as follows::
```

Comment:
Maybe we could keep a consistence, `./python -m test -v test_that_failed` or `make test TESTOPTS="-v" test_that_failed`